apps/server/src/modules/oauth/service/dto/oauth.response.ts

```diff
@@ -20,4 +20,14 @@ export class OAuthResponse {
 	@IsString()
 	@IsNotEmpty()
 	logoutEndpoint?: string;
+
+	@IsOptional()
+	@IsString()
+	@IsNotEmpty()
+	provider?: string;
```

Comment:
Really optional?

Reply:
Good point. made provider AND redirect mandatory.